packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -798,7 +794,7 @@ describe("ServerlessHub.js", function () {
     // Logs should include correct starting and latest block numbers for the alternate network.
     const alternateBlockNumbers = {
       [alternateChainId]: {
-        lastQueriedBlockNumber: 0,
+        lastQueriedBlockNumber: latestAlternateBlockNumber, // Same as latest as this is first time bot is run.
```

Comment:
Why are they the same the first time the bot is run?

Reply:
otherwise the first run would cover all events since genesis.